google-cloud-tools-plugin/resources/messages/CloudToolsBundle.properties

```diff
@@ -468,3 +389,5 @@ cloudsdk.check.for.updates.action=Check for updates now
 cloudsdk.location.browse.window.title=Browse to Cloud SDK Directory
 cloudsdk.location.missing.message=Cloud SDK home directory is not specified.
 cloudsdk.download.message={0}Click here{1} to download the Cloud SDK.
+
+clouddebug.logout.additional.message=Any Stackdriver Debug sessions listening in the background will be stopped.
```

Comment:
is this is a mistake? (I assume you meant to remove it)

Reply:
no.   It's used by the the login extender which stays in that module